tests/foreman/cli/test_reporttemplates.py

```diff
@@ -758,3 +841,93 @@ def test_positive_generate_ansible_template(self):
         })
 
         assert host['name'] in [item.split(',')[1] for item in report_data if len(item) > 0]
+
+    @tier3
+    def test_positive_generate_entitlements_csv_report(self):
+        """Generate a cvs report using the Entitlements template.
+
+        :id: f722f995-2bdc-462c-b8be-5fa8ff999a9f
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, and Subscriptions.
+
+        :steps:
+            1. hammer report-template generate --organization '' --id '' --report-format ''
+
+        :expectedresults: csv report is generated containing all the expected information
+                          regarding Entitlements.
+
+        :CaseImportance: High
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.install_katello_ca()
+            vm.register_contenthost(self.setup_org['label'], self.setup_new_ak['name'])
+            assert vm.subscribed
+            result = ReportTemplate.generate({
+                'organization': self.setup_org['name'],
+                'id': 115,
+                'report-format': 'csv'
+            })
+            assert vm.hostname in result[1]
+            assert self.setup_subs_id[0]['name'] in result[1]
```

Comment:
What are these `result[1]`? Is it possible to rewrite that, so small change in output format doesn't fail the test?

Reply:
results[1] is just a list of values, I can loop through instead of using indexes.

Reply:
After looking at this again, I do not think It is worth the effort to change this since the return structure should remain static.